src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/unit_tests/controllers/api/ReportRunControllerTests.kt

```diff
@@ -78,8 +92,9 @@ class ReportRunControllerTests : ControllerTest()
 
         val mockAPIResponse = OrderlyServerResponse(mockAPIResponseText, 200)
 
+        val expectedBody = "{\"params\":{}}"
```

Comment:
This is maybe more readable and easier to type, especially if things get a bit more complicated. But absolutely no need to change it!
```suggestion
        val expectedBody = """{"params":{}}"""
```